docker/prod/Dockerfile

```diff
@@ -1,13 +1,31 @@
 ARG RUST_BUILDER_IMAGE=ekidd/rust-musl-builder:stable
 
-FROM $RUST_BUILDER_IMAGE as rust
+FROM $RUST_BUILDER_IMAGE as planner
+WORKDIR /app
+RUN sudo chown -R rust:rust .
+# We only pay the installation cost once, 
+# it will be cached from the second build onwards
```

Comment:
In CI it will have to be compiled every time (unless you plan to cache the entire docker image).